cypress/integration/researchGroupCanSignUp.spec.js

```diff
@@ -0,0 +1,52 @@
+describe("Research Group Sign-Up", () => {
+  beforeEach(function() {
```

Comment:
You don't need to pull the articles before each test, or arguably before any of the tests.

Reply:
Good catch

Reply:
Yes, you do. Articles are fetched every time we load the app index page.

Reply:
@tochman reverting thanks.